.pylintrc

```diff
@@ -32,7 +32,7 @@ load-plugins=
 #R0903 = Too few public methods (%s/%s)
 #C0326 = bad-whitespace
 #I0011 = locally-disabling
-disable=R0903,C0326,I0011,redefined-variable-type,c-extension-no-member
+disable=R0903,C0326,I0011,redefined-variable-type,c-extension-no-member,W,C,R
```

Comment:
We only ever run `pylint -E` in CI so this does the same. At some point we should reassess these settings and clean-up a lot of useless `# pylint: disable=xxxx` comments but that's a task for another day...

Reply:
We run `pylint -E` because those are things that need to be fixed because they are broken.  (apart from pylint acting strangely)
We cannot run without `-E` because there are too many things being reported otherwise. (unless we can run pylint on a diff)
The .pylintrc isn't only for the CI though. Developers should also take care of fixing warnings (maybe at least be aware) when they develop new things, so we shouldn't disable all of them in the standard pylint config.

Reply:
Done